### DISCUSSION TITLE
Co-Registering 3D data

### DISCUSSION THREAD

**Discussion by CometManAtGitHub** [MIC-DKFZ/nnUNet#683] [Q&A] [answered]
Hi,

you state in "Dataset conversion instructions", that the data has to be co-registered

"Also make sure that all your data is co-registered!"
https://github.com/MIC-DKFZ/nnUNet/blob/master/documentation/dataset_conversion.md

Do you mean it is required co-register the various modalities for each patient, when different modalities are used?
Or do we have to co-register all images?

Thanks.

**Comment by FabianIsensee** [comment, marked as answer]
For each case, modalities need to be coregistered. Cases do not need to be coregistered to each other, that would make no sense ;-)
Best,
Fabian
